tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 360x360 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#let's do some posterior predictive checks to see if the\n",
+    "#observation `x_o` remains within the support of the posterior\n",
+    "from sbi.analysis import pairplot\n",
+    "\n",
+    "fig, ax = pairplot(\n",
+    "    samples=posterior_data,\n",
+    "    points=x_o,\n",
+    "    limits=list(zip(.75*x_o.flatten(), 1.25*x_o.flatten())),\n",
+    "    upper=['kde'],\n",
+    "    diag=['kde'],\n",
+    "    figsize=(5,5),\n",
+    "    labels=[rf\"$\\theta_{d}$\" for d in range(3)]\n",
```

Comment:
this is confusing, a posterior predictive plot is always in data space, but we theta labels here.

Reply:
:face_with_head_bandage: true. Fixed with 6cc72a4